release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -148,6 +148,50 @@ int result = -9223372036854775808; // error: '9223372036854775808' is out of ran
   - `createEnumDeclarationNode`
   - `createClassDefinitionNode`
 
+- Restricted the type-binding for query expression outer join to be inferred as a nillable type 
+    Earlier, the type binding for outer join could be any type-descriptor but now it's restricted to be 'var' so that it will be inferred as a nillable type.
```

Comment:
```suggestion
- Restricted the typed binding for query expression outer join to be inferred as a nillable type. Earlier, the type binding for outer join could be any type descriptor but now it's restricted to be `var` so that it will be inferred as a nillable type.
```

Reply:
Not sure which is technically correct `typed binding` or `type binding`. Shall we check in the spec?

Reply:
```suggestion
- Restricted the typed binding for query expression outer join to be inferred as a nillable type. Earlier, the typed binding for outer join could be any type descriptor but now it's restricted to be `var` so that it will be inferred as a nillable type.
```